test/plugin/base_test.rb

```diff
@@ -1609,21 +1667,24 @@ def verify_json_log_entries(n, params, payload_type = 'textPayload')
         labels.merge!(entry['labels'] || {})
 
         assert_equal \
-          "projects/#{params[:project_id]}/logs/#{params[:log_name]}", log_name
+          "projects/#{params[:project_id]}/logs/#{params[:log_name]}",
+          log_name if params[:log_name]
```

Comment:
I suspect this is what rubocop made you do, but personally I find this pretty unreadable -- the `if` applies to the whole `assert_equal` statement, but it *seems* that it only applies to `log_name` (making it effectively `assert_equal "...", nil` when `params[:log_name]` is `nil`).
It would be much more obvious as a conditional statement:
```ruby
        if params[:log_name]
          assert_equal \
            "projects/#{params[:project_id]}/logs/#{params[:log_name]}", log_name
        end
```
IMO, this is a good case for disabling this specific rubocop check for this line (`# rubocop:disable Style/GuardClause`).

Reply:
Changed. Rubocop did not complain about it when it multiple lines actually. :)